packages/functions/src/index.node.ts

```diff
@@ -15,9 +15,8 @@
  * limitations under the License.
  */
 import { registerFunctions } from './config';
-import { fetch as undiciFetch } from 'undici';
 
 export * from './api';
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-registerFunctions(undiciFetch as any, 'node');
+registerFunctions('node');
```

Comment:
Hm, I don't think we actually need a separate functions Node bundle anymore. Maybe that can be a separate PR though. The only thing different is that it labels the bundle "node" for platform logging, which is pointless if it's the same bundle. I guess it tells us how many node users are using functions. Also it doesn't export the types from public-types which is probably a mistake.

Reply:
I'll add the export.

Reply:
We spoke offline about removing the separate node bundle, will be done in a different PR.